test/test_mxnet.py

```diff
@@ -44,8 +44,11 @@ def test_horovod_allreduce(self):
         """Test that the allreduce correctly sums 1D, 2D, 3D tensors."""
         hvd.init()
         size = hvd.size()
-        dtypes = ['int32',   'int64',
-                  'float32', 'float64']
+        if 'MLSL_ROOT' in os.environ:
```

Comment:
Please use the same `filter_supported_types` pattern here for consistency.

Reply:
Done